src/docs/sphinx/blueprint_mesh.rst

```diff
@@ -504,12 +516,85 @@ The following diagram illustrates a simple **multi-buffer** material set example
       # +--------+--------+--------+
       #
 
+      matsets:
+        matset:
+          topology: topology
+          volume_fractions:
+            a:
+              values: [0, 0, 0, a1, 0, a0]
+              indices: [5, 3]
+            b:
+              values: [0, b0, b2, b1, 0]
+              indices: [1, 3, 2]
+
+
+Material Set Indexing Variants
+=================================
+
+Another dimension along which material sets can vary is in how their volume fractions/topological element association is established.
+This dimension produces two additional material set variants: **element-dominant** association (element indices control volume fraction order) and **material-dominant** association (material indices control volume fraction order).
+Both of these variants and their corresponding schemas are outlined in the subsections below.
+
+
+Element-Dominant Material Sets
+*********************************
+
+An **element-dominant** material set is one that orders its volume fraction data so that it matches the topological element ordering.
```

Comment:
Perhaps rephrase to "In an **element-dominant** material set, volume fraction data order matches topological element order."